src/container/container_iv.c

```diff
@@ -653,7 +653,7 @@ cont_iv_capability_update(void *ns, uuid_t cont_hdl_uuid, uuid_t cont_uuid,
 
 	rc = cont_iv_update(ns, IV_CONT_CAPA, cont_hdl_uuid, &iv_entry,
 			    sizeof(struct cont_iv_entry),
-			    CRT_IV_SHORTCUT_TO_ROOT, CRT_IV_SYNC_EAGER);
+			    CRT_IV_SHORTCUT_TO_ROOT, CRT_IV_SYNC_LAZY);
```

Comment:
please see https://jira.hpdd.intel.com/browse/DAOS-2787 , it caused quite some trouble for CI a few days ago when move some tests to VM in CI.
if you want to make CAPA distribution be async, may need move the ds_cont_tgt_open out from IV path.

Reply:
ah, let me check. Thanks